run-tests.sh

```diff
@@ -307,6 +307,9 @@ donot '@dataguids'
 # Do not run prjsBucketAccess (for prod-execution only)
 donot '@prjsBucketAccess'
 
+# run the test until argo works in jenkins envs
+donot '@GWASUI'
```

Comment:
trying to understand this line here, will this disable the UI tests until argo works in jenkins?